mobile_wallet/scripts/build-ios.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+(cargo build --manifest-path ./Cargo.toml --release --target x86_64-apple-ios && \
+cargo build --manifest-path ./Cargo.toml --release --target aarch64-apple-ios-sim && \
+cargo build --manifest-path ./Cargo.toml --release --target aarch64-apple-ios)
```

Comment:
This is not super important, but we don't need the outer `(...)` and the `&& \` either.